x/zoneconcierge/keeper/ibc_packet_btc_headers.go

```diff
@@ -0,0 +1,41 @@
+package keeper
+
+import (
+	"context"
+
+	"github.com/babylonlabs-io/babylon/x/zoneconcierge/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// BroadcastBTCHeaders sends an IBC packet of BTC headers to all open IBC channels to ZoneConcierge
+func (k Keeper) BroadcastBTCHeaders(ctx context.Context) {
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+
+	openZCChannels := k.GetAllOpenZCChannels(ctx)
+	if len(openZCChannels) == 0 {
+		k.Logger(sdkCtx).Info("no open IBC channel with ZoneConcierge, skip broadcasting BTC headers")
+		return
+	}
+
+	// TODO: currently sending last w+1 headers but should fetch from BSN base header to tip
```

Comment:
Is this something planned for the future and do we have task for it ?

Reply:
Reading the code again -- does `headers` always contain w+1 headers or only new headers since the last segment? My understanding is the latter but not former